### DISCUSSION TITLE
GDC question

### DISCUSSION THREAD

**Discussion by jcsm89** [leondgarse/Keras_insightface#94] [Q&A] [answered]
Hey! First of all thanks for all the hard work you've been putting on this repository, it has been very helpful! What exactly is the GDC output layer? I've googled about a little bit and came upon the "GDC block: gradient-guided direction-aware convolution 
block for image classification" paper, which seems to fit the bill, but I'm not sure if this is it. If it is, what is the rational of using this instead of a simple fully conneceted? Any obvious advantage?

**Comment by jcsm89** [comment, marked as answer]
The paper I linked above seems really unrelated, which leaves with the question of what this GDC is at all :) Might be something very obvious that I can't ascertain

**Reply by jcsm89** [reply]
If anyone has this question I found the answer: https://stackoverflow.com/questions/56163778/what-is-a-global-convolution-in-cnns
Sorry for the unnecessary discussion but now it's visible for anyone that has the same question :)

**Reply by leondgarse** [reply]
Ya, that's it. :)
Generally, it's using `DepthwiseConv2D` instead of the commonly used `GlobalAveragePooling2D` layer.
